src/Clients/CaradhrasLimitsClient.php

```diff
@@ -35,6 +35,14 @@ public function getAccountLimits(string $accountId)
             ->object();
     }
 
+    public function getAccountLimitRequests(string $accountId)
+    {
+        return $this
+            ->apiClient()
+            ->get("/limits/v2/requests?idAccount={$accountId}")
```

Comment:
podemos chamar dessa forma tbm, invés de fazer manualmente a query

```suggestion
            ->get("/limits/v2/requests", ['idAccount' => $accountId])
```